ejb3/src/main/java/org/jboss/as/ejb3/subsystem/EJBTransformers.java

```diff
@@ -301,4 +320,59 @@ private static void registerPassivationStoreTransformers_1_2_1_and_1_3_0(Resourc
                 .setValueConverter(AttributeConverter.Factory.createHardCoded(new ModelNode().set(TimeUnit.SECONDS.name()), true), EJB3SubsystemModel.IDLE_TIMEOUT_UNIT)
         ;
     }
+
+    /**
+     * This Combined Transformer manages this transformation from EAP7 to previous versions (attribute definition/xml markup):
+     *
+     * DEFAULT_SFSB_CACHE / cache-ref --> DEFAULT_CLUSTERED_SFSB_CACHE / clustered-cache-ref
+     * DEFAULT_SFSB_PASSIVATION_DISABLED_CACHE / passivation-disabled-cache-ref --> DEFAULT_SFSB_CACHE / cache-ref
+     *
+     */
+    private static class SfsbChacheRefTransformer implements CombinedTransformer {
+        private final Map<String, String> renames;
+
+        public SfsbChacheRefTransformer(){
+            renames = new HashMap<>();
+            renames.put(DEFAULT_SFSB_CACHE, EJB3SubsystemModel.DEFAULT_CLUSTERED_SFSB_CACHE);
+            renames.put(DEFAULT_SFSB_PASSIVATION_DISABLED_CACHE, EJB3SubsystemModel.DEFAULT_SFSB_CACHE);
+        }
+
+        @Override
+        public TransformedOperation transformOperation(TransformationContext context, PathAddress address, ModelNode operation) throws OperationFailedException {
+            if (Operations.getName(operation).equals(ADD)){
```

Comment:
The only requirement I have found to add here the ADD operation is due to the use of the transform in the test. I could find out a case wherein a mixed domain we have to add the EJB3 subsystem, maybe if as a user I remove the whole subsystem to add it later. Do you know another example that requires this operation added to the transform?